CorsixTH/Lua/objects/plant.lua

```diff
@@ -70,11 +70,13 @@ object.orientations = {
 -- For litter: put broom back 356
 -- take broom out: 1874
 -- swoop: 1878
--- For plant: droop down: 1950
--- back up again: 1952
+-- Frames for plant states are
```

Comment:
I didn't mean animation numbers.... ok, maybe you should just document self.current_state (which was an index into the state array)